README.md

```diff
@@ -17,6 +17,24 @@ The repository contains two directories:
 - `packages` containing a cloudflare package that can be used to build a Cloudflare Workers-compatible output for Next.js applications.
 - `examples` containing Next.js applications that use the above mentioned cloudflare package.
 
-### How to try out/develop in the repository
+### How to try out the `@opennextjs/cloudflare` package
+
+You can simply install the package from npm as specified in the [OpenNext documentation](https://opennext.js.org/cloudflare/get-started).
+
+> [!note]
+> There are also prerelease packages that we automatically publish on branch pushes (using [`pkg.pr.new`](https://github.com/stackblitz-labs/pkg.pr.new)):
+>
+> - `https://pkg.pr.new/@opennextjs/cloudflare@main`:
+>
+>   Updated with every push to the `main` branch, this prerelease contains the most up to date yet (reasonably) stable version of the package.
+>
+> - `https://pkg.pr.new/@opennextjs/cloudflare@experimental`
+>
+>   Updated with every push to the `experimental` branch (when present), this prerelease contains the latest experimental version of the package (containing features
+>   that we want to test/experiment on before committing to).
+>
+> Which you can simply install directly with your package manager of choice, for example: `npm i https://pkg.pr.new/@opennextjs/cloudflare@main`
```

Comment:
```suggestion
> Which you can simply install directly with your package manager of choice, for example: `npm i https://pkg.pr.new/@opennextjs/cloudflare@experimental`
```

Reply:
As per my comment in the aws repo: https://github.com/opennextjs/opennextjs-aws/pull/614#discussion_r1825647410

I feel like using `main` in the example command makes more sense and this is how it looks:
![Screenshot 2024-11-01 at 09 56 51](https://github.com/user-attachments/assets/dd09c22e-053b-495c-92fe-06076629a8b3)

hopefully this is fine? 🙂